### PR TITLE
Enlarge bundle export

### PR DESCRIPTION
**Pull request by bargoldi**

### CODE REVIEW

File: src/services/index.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export * from './basic-desc/basic-desc.service';
```
Comment (bargoldi) [inline src/services/index.ts]: I'll remove the layer services, but basic-desc and basic-static-primitives are extensible this way.

File: src/components/ac-layer/ac-layer.component.ts [hunk 2]
```diff
@@ -16,6 +16,8 @@ import { StaticPolylineDrawerService } from '../../services/static-polyline-draw
 import { PolygonDrawerService } from '../../services/polygon-drawer/polygon-drawer.service';
 import { ArcDrawerService } from '../../services/arc-drawer/arc-drawer.service';
 import { PointDrawerService } from '../../services/point-drawer/point-drawer.service';
+import 'rxjs/add/observable/merge';
```
Comment (eitanfr) [inline src/components/ac-layer/ac-layer.component.ts]: checkout `src/operators.ts` that i added recently , where operators are loaded globally

File: src/services/index.ts (hunk 1 above)
Comment (ttsahi) [inline src/services/index.ts]: Ok, its irelevant because there is no api to extend the framework.

Comment (eitanfr) [inline src/services/index.ts]: @ttsahi means that even if you inherit from the basic class you still need to change the implementation inside ac-layer. so for the current state you can't create a freaky primitive , we need to think about it if we want to add this option in the future.

Comment (bargoldi) [inline src/services/index.ts]: Hmm got the 'irrelevant' idea.
Changing it.

**Pull request merged**